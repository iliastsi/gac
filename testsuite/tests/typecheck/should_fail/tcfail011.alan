foo() : proc
  a : int [-15];
  b : int [-1000000000000000];
  c : int [0];
  d : int [10000000000000000];
{
  return;
}
