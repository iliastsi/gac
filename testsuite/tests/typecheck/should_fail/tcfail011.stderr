tcfail011.alan:2:3: Error: Array definition error at `a : int[-15];'
	Array size must be a positive integer

tcfail011.alan:3:3: Error: Array definition error at `b : int[-1000000000000000];'
	Array size must be a positive integer

tcfail011.alan:4:3: Error: Array definition error at `c : int[0];'
	Array size must be a positive integer

tcfail011.alan:5:12: Warning: Overflow in implicit constant conversion at `10000000000000000'
	Integer has been rounded to `2147483647'

